charts/rasa-x/templates/rasa-open-source-ingress.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if and .Values.ingress.enabled (not .Values.nginx.enabled) -}}
+{{- $fullName := include "rasa-x.fullname" . -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}-rasa-open-source
+  labels:
+    {{- include "rasa-x.labels" . | nindent 4 }}
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+  {{- with .Values.ingress.annotations }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+{{- if .Values.ingress.tls }}
+  tls:
+  {{- range .Values.ingress.tls }}
+    - hosts:
+      {{- range .hosts }}
+        - {{ . | quote }}
+      {{- end }}
+      secretName: {{ .secretName }}
+  {{- end }}
+{{- end }}
+  rules:
+  {{- range .Values.ingress.hosts }}
+  - host: {{ .host | quote }}
+    http:
+      paths:
+      - backend:
+          serviceName: {{ $fullName }}-{{ $.Values.rasa.versions.rasaProduction.serviceName }}
+          servicePort: {{ $.Values.rasa.port }}
+        path: /core/(.*)
+      - backend:
+          serviceName: {{ $fullName }}-{{ $.Values.rasa.versions.rasaProduction.serviceName }}
+          servicePort: {{ $.Values.rasa.port }}
+        path: /webhooks/
+      - backend:
+          serviceName: {{ $fullName }}-{{ $.Values.rasa.versions.rasaProduction.serviceName }}
```

Comment:
I think this might need the `proxy_http_version 1.1;` part for the sockets (at least the `/ws` websockets didn't work for me otherwise).

Reply:
HTTP 1.1 version is the default for nginx-ingress